dftimewolf/lib/preflights/cloud_token.py

```diff
@@ -37,4 +40,38 @@ def CleanUp(self):
     """Carries out optional cleanup actions at the end of the recipe run."""
 
 
-modules_manager.ModulesManager.RegisterModule(GCPTokenCheck)
+class AWSAccountCheck(module.PreflightModule):
+  """Checks for AWS authentication."""
+
+  def SetUp(self, profile_name=None):  # pylint: disable=arguments-differ
+    """Runs a boto3 check to ensure AWS authentication is configured
+
+    Args:
+      profile_name (str): Optional. The profile to test.
```

Comment:
What happens if I don't specify the profile? What gets tested?

Reply:
The default profile (as resolved by boto3) - have clarified in docstring.